kotest-tests/kotest-tests-core/src/jvmTest/kotlin/com/sksamuel/kotest/specs/funspec/FunSpecWithContextBlockTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.sksamuel.kotest.specs.funspec
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.collections.shouldContainExactly
+import io.kotest.matchers.shouldBe
+import kotlinx.coroutines.delay
+
+// Verifies fix for : https://github.com/kotest/kotest/issues/2710
+class FunSpecWithContextBlockTest : FunSpec({
```

Comment:
Can we call the test FunSpecConfigEnabledTest and test all the enabled combinations ?